packages/block-editor/src/components/inner-blocks/use-nested-settings-update.js

```diff
@@ -25,6 +25,7 @@ const pendingSettingsUpdates = new WeakMap();
  * @param {string}               clientId                   The client ID of the block to update.
  * @param {string[]}             allowedBlocks              An array of block names which are permitted
  *                                                          in inner blocks.
+ * @param {string[]}             prioritizedInserterBlocks  Block names and/or block variations to be prioritized in the inserter.
```

Comment:
We should be a bit more specific how a block variation is expressed in this API, and this applies to the docs above. We use a convention of `{blockName}/{variationName}` due to the inserter [items structure](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/store/selectors.js#L1816)